paas/install_client.go

```diff
@@ -150,6 +174,12 @@ func (c *InstallClient) Uninstall(cmd *cobra.Command) error {
 	return nil
 }
 
+// InstallDeployment installs one single Deployment on the cluster
+func (c *InstallClient) InstallDeployment(deployment kubernetes.Deployment, logger logr.Logger) error {
```

Comment:
I will use this method to install minibroker (and other brokers) as separate carrier subcommands (in future PRs).